strings-csharp/CheckNumberInString/CheckNumberInStringTests/GlobalUsings.cs

```diff
@@ -0,0 +1,3 @@
+global using CheckNumberInString;
+
```

Comment:
These blank lines can be removed. Probably wouldn't have bothered mentioning it, but since there's the other `var` keyword to change anyway...